zebra-chain/src/block/tests.rs

```diff
@@ -165,31 +171,25 @@ fn block_limits_multi_tx() {
         .zcash_serialize(&mut data)
         .expect("block should serialize as we are not limiting generation yet");
 
-    // Check size
-    assert_eq!(data.len(), 1999925);
-
     // Deserialize by now is ok as we are lower than the limit
     let block2 = Block::zcash_deserialize(&data[..])
         .expect("block should deserialize as we are just below limit");
 
     assert_eq!(block, block2);
-    assert_eq!(limit_txs, block2.transactions.len());
+    assert_eq!(max_transactions_in_block, block2.transactions.len());
 
     // Add 1 more transaction to the block, limit will be reached
-    limit_txs = limit_txs + 1;
+    max_transactions_in_block = max_transactions_in_block + 1;
     many_transactions.push(Arc::new(tx));
     block.transactions = many_transactions.clone();
-    assert_eq!(limit_txs, many_transactions.len());
+    assert_eq!(max_transactions_in_block, many_transactions.len());
 
     // Serialize will still be fine
     let mut data = Vec::new();
     block
         .zcash_serialize(&mut data)
         .expect("block should serialize as we are not limiting generation yet");
 
-    // Check size
-    assert_eq!(data.len(), 2000010);
```

Comment:
Issue: The block size is the purpose of the test, so let's make sure `data.len() > MAX_BLOCK_BYTES`.